.ci/install_kata_image.sh

```diff
@@ -109,8 +109,14 @@ build_image() {
 
 	pushd "${osbuilder_path}" >/dev/null
 
-	readonly ROOTFS_DIR="${PWD}/rootfs"
-	export ROOTFS_DIR
+	# Verify rootfs dir
+	if [ "${TEST_CGROUPSV2}" == "true" ] && [ "${TEST_INITRD}" == "yes" ]; then
+		distro_name=$(echo "${distro}" | sed 's/./\u&/')
+		ROOTFS_DIR="${osbuilder_path}/rootfs-builder/rootfs-${distro_name}"
+	else
+		readonly ROOTFS_DIR="${PWD}/rootfs"
+		export ROOTFS_DIR
```

Comment:
Shouldn't this `export` be used after the end of the `if`/`else` test so it applies in all scenarios?